subsys/bluetooth/host/conn.c

```diff
@@ -1959,6 +1989,74 @@ struct bt_conn *bt_conn_create_le(const bt_addr_le_t *peer,
 	return conn;
 }
 
+#if defined(CONFIG_BT_WHITELIST)
+int bt_le_autoconn_add(const bt_addr_le_t *addr)
```

Comment:
We've discussed this internally and we believe this new API is too focused on central. Having instead a set of functions to deal with whitelist:

`bt_le_wl_add()`
`bt_le_wl_remove()`

and then adding additional parameters to:

`bt_conn_create_le()`
and
`bt_le_adv_start()`
and
`bt_le_scan_start()`

would make more sense so that the use of the whitelist is generalized to all roles.